tests/generators/unittest_dart.js

```diff
@@ -154,3 +157,22 @@ Blockly.Dart['unittest_fail'] = function(block) {
         '}']);
   return functionName + '(' + message + ');\n';
 };
+
+
+Blockly.Dart['unittest_adjustindex'] = function(block) {
+  var index = Blockly.Dart.valueToCode(block, 'INDEX',
+                Blockly.Dart.ORDER_ADDITIVE) || '0';
```

Comment:
Strange indenting here.
Also, there are two blank lines in front of this function whereas other functions only have one.